x-pack/plugins/ml/public/application/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -145,27 +175,25 @@ export const AnalyticsPanel: FC<Props> = ({
                 </h3>
               </EuiText>
             </EuiFlexItem>
-            {analyticsStats !== undefined && (
-              <EuiFlexItem grow={false} className="mlOverviewPanel__statsBar">
-                <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
-              </EuiFlexItem>
-            )}
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup gutterSize={'s'} alignItems="center">
+                {analyticsStats !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <StatsBar stats={analyticsStats} dataTestSub={'mlOverviewAnalyticsStatsBar'} />
+                  </EuiFlexItem>
+                ) : null}
+                <EuiFlexItem grow={false}>
+                  <EuiButton size="m" fill href={manageJobsLink}>
+                    {i18n.translate('xpack.ml.overview.analyticsList.manageJobsButtonText', {
+                      defaultMessage: 'Manage jobs',
+                    })}
+                  </EuiButton>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiFlexItem>
           </EuiFlexGroup>
           <EuiSpacer />
           <AnalyticsTable items={analytics} />
```

Comment:
The `Creation time` column is getting truncated, although it looks like there is room available for the full time:

![image](https://user-images.githubusercontent.com/7405507/146041794-ac177231-5dd5-4d90-9fad-c9225918d23b.png)

Reply:
Fixed in b2dcb531e2b849a57fd5a0a204b0027cf79a7d10

Reply:
As for the Anomaly detection jobs table, I wonder if we need to format down to the second level here on the Overview page?

Reply:
Done in 84caa57dc55070fc2e778512b214f130d04506ca